build-linux-appimage.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# This script assumes $PWD is the same dir in which this script is located
+
+# Helps avoid permissions problems with `jenkins` user in docker container when
+# making a local packaged build
+git clean -dfx
+
+# Build .AppImage
+docker run -it --rm \
```

Comment:
Could we maybe move these scripts into `scripts`? The root of this repos is already cluttered enough as it is.